burst-kotlin-plugin/src/main/kotlin/app/cash/burst/kotlin/Argument.kt

```diff
@@ -58,5 +61,13 @@ internal fun IrPluginContext.allPossibleArguments(
   val referenceClass = referenceClass(classId)?.owner ?: return null
   if (!referenceClass.isEnumClass) return null
   val enumEntries = referenceClass.declarations.filterIsInstance<IrEnumEntry>()
-  return enumEntries.map { Argument(parameter, parameter.type, it) }
+  val defaultValueSymbol = (parameter.defaultValue?.expression as? IrGetEnumValue)?.symbol
```

Comment:
We talked about erroring if the default value is anything other than a constant enum value. Otherwise you could silently refactor out common defaults into a helper function (or something) and silently break the specialization.

Reply:
Lemme do that in a follow-up!

Reply:
https://github.com/cashapp/burst/issues/37